### DISCUSSION TITLE
installing python

### DISCUSSION THREAD

**Discussion by kmjsmith** [codeproject/CodeProject.AI-Server#147] [Q&A] [answered]
hi the latest codeproject ai seems to have an issue installin python 37 when installin YOLO

![Screenshot 2024-10-28 221338](https://github.com/user-attachments/assets/d1de837d-71fe-4cd8-9b54-3d0b6fc60efd)

**Comment by tbc725** [comment, marked as answer]
Try to reinstall again as it seems to have some changes recently such as the link. Maybe that handles it.
